docs/getting-started-guides/windows/README.md

```diff
@@ -0,0 +1,163 @@
+# Running Windows Server Containers using Kubernetes
+Microsoft, in collaboration with Docker, is introducing a new feature called Windows Server Containers with Windows Server 2016, which enables Docker containers to run on Windows. We have enhanced Kubernetes to support Windows Server Containers (and Windows Server 2016 as a container host node).
```

Comment:
Kubernetes documentation typically does not highlight the work of a specific company, group or individual.  (A blog post is a great place for giving credit.)  

With that in mind, I'd suggest a couple of rewordings:

The first sentence could be reworded as:

I'd suggest replacing "we" everywhere in this doc.  For example,   just say "Kubernetes supports Windows Server Containers ...." 

I'd also suggest rewording the first two sentences to  something like:

"Kubernetes supports nodes running Microsoft Windows Server 2016.  Docker version x.xx is required .  Pods are comprised of Windows Server Containers."  

You can definitely link from various terms (e.g. Windows Server Containers) to product pages on the Microsoft and Docker websites.